specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -478,6 +478,122 @@
         }
       }
     },
+    "/providers/Microsoft.ADHybridHealthService/addsservices/{serviceName}/features/{featureName}/userpreference": {
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets the user preferences for a given feature.",
+        "x-ms-examples": {
+          "addsServices_getUserPreference": {
+            "$ref": "./examples/UserPreference.json"
+          }
+        },
+        "operationId": "addsServices_getUserPreference",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "featureName",
+            "in": "path",
+            "description": "The name of the feature.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/UserPreference"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [ "Adds" ],
+        "description": "Deletes the user preferences for a given feature.",
+        "x-ms-examples": {
+          "addsServices_deleteUserPreference": {
+            "$ref": "./examples/DeleteUserPreference.json"
+          }
+        },
+        "operationId": "addsServices_deleteUserPreference",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "featureName",
+            "in": "path",
+            "description": "The name of the feature.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {

```

Comment:
Please provide meaningful description for all operations
Please confirm whether this is a long running operation

Reply:
@dsgouda, did not get your point. What is a meaningful description for an operation which does not display any result? Do you have an example for me to follow?

Reply:
@dsgouda , also will be good if you can help me with an example for long running operation.

Reply:
Here is an example. 
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2018-05-01/applicationGateway.json#L72

Reply:
So none of the operations listed in my swagger spec are long running.
Do I still need to add that flag? All the delete, add,get are targetted operations with a max latency of 6s.

Reply:
This one I left out, waiting for the clarification for my previous qs.

Reply:
Given the latency you do not need to set the flag.